mkdocs.yml

```diff
@@ -14,9 +14,10 @@ nav:
       - Document Search:
           - how-to/document_search/async_processing.md
           - how-to/document_search/create_custom_execution_strategy.md
-          - how-to/document-search/search_documents.md
-          - how-to/document-search/use_rephraser.md
-          - how-to/document-search/use_reranker.md
+          - how-to/document_search/search_documents.md
```

Comment:
honestly, I'd prefer to change it to a dashes everywhere, because it'll be the URL for those paths and the dashes are favored by Google - https://developers.google.com/search/docs/crawling-indexing/url-structure?hl=en&visit_id=638684779403302040-2044200795&rd=1